.github/scripts/update-platform-executables.js

```diff
@@ -1,20 +1,29 @@
 const core = require('@actions/core');
 const { GitHub } = require('@actions/github');
 const fs = require('fs');
+const path = require('path');
 
 /**
  * Functionality from svenstaro/upload-release-action.
  * Link: https://github.com/svenstaro/upload-release-action
  */
 const updatePlatformExecutables = async () => {
   try {
-    const requiredParams = ['GITHUB_TOKEN', 'REPO_NAME', 'TAG_NAME', 'ASSET_NAME', 'FILE'];
+    const requiredParams = ['GITHUB_TOKEN', 'REPO_NAME', 'TAG_NAME'];
     for (const param of requiredParams) {
       if (!(param in process.env)) {
         core.setFailed(`Exiting, required param: ${param} is missing`);
         return;
       }
     }
+    if(('FILE' in process.env) && !('ASSET_NAME' in process.env)){
+      core.setFailed(`Exiting, required param: ASSET_NAME is missing`);
+      return;
+    }
+    if(!('FILE' in process.env) && !('DIRNAME' in process.env)){
```

Comment:
Should this be an `OR` condition / `AND` ? The error message says `OR`.

Reply:
Need to modify the error message.